WooCommerce/Classes/ViewRelated/Dashboard/Settings/Help/ApllicationLogDetailView.swift

```diff
@@ -0,0 +1,106 @@
+import SwiftUI
+
+/// Hosting controller that wraps an `ApplicationLogDetailView`
+///
+final class ApplicationLogDetailViewController: UIHostingController<ApplicationLogDetailView> {
+    init(viewModel: ApplicationLogViewModel) {
+        super.init(rootView: ApplicationLogDetailView(viewModel: viewModel))
+        self.title = viewModel.title
+        viewModel.present = { [weak self] vc in
+            self?.present(vc, animated: true, completion: nil)
+        }
+    }
+
+    required dynamic init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+}
+
+struct ApplicationLogDetailView: View {
+    @ObservedObject var viewModel: ApplicationLogViewModel
+
+    var body: some View {
+        ScrollViewReader { scrollProxy in
+            List(viewModel.lines) { line in
+                VStack(alignment: .leading, spacing: 6) {
+                    line.dateText
+                        .map {
+                            Text($0)
+                                .footnoteStyle()
+                        }
```

Comment:
I think this can become
```suggestion
                    Text(line.dateText)
                        .footnoteStyle()
```
Or did I miss something? 😅

Reply:
`dateText` is an optional so that won't work. However, I think I got confused about what syntax is allowed in a result builder, and a regular `if let` seems to work, and it's clearer. I replaced it in c8a33d739